Adding Attenuation Model of Ghasemi (2009)

FatemehAlsh opened the pull request:
Attenuation model of Ghasemi (2009) is developed for Iran with the doi of 10.1007/s10950-008-9143-x.
A gsim library of this model is added. 
The version and tag of openquake engine used to develope this library is 3.17.1.

The changes made are: 
1- A gsim library of "ghasemi_2009.py" is added to openquake\hazardlib\gsim.
2- A test file of "ghasemi_2009_test.py" is added to openquake\hazardlib\tests\gsim 
3- A verification table of "Ghasemi09.xlsx" is added to openquake\hazardlib\tests\gsim\data. 
     The verification is done using an excel file which calculates the Sa(T1) based on model developed by Ghasemi2009. The 
     results obtained from this excel file is compared with the ones obtained from oq. 
4- The "openquake.hazardlib.gsim.rst" file is modified so that it includes "ghasemi_2009" gsim library.

micheles commented:
The file ghasemi_2009_test.py does not contain tests, I guess you uploaded the wrong file by mistake,

FatemehAlsh commented:
> The file ghasemi_2009_test.py does not contain tests, I guess you uploaded the wrong file by mistake.

You're right. I'd made a mistake. I replaced the wrong file with the correct ghasemi_2009_test.py one. Also, I noticed I had put the .xlsx format of test table file instead of .csv one, so I modified this too. Moreover, the name of the main class of ghasemi_2009 library is modified so it is "GhasemiEtAl2009" now.

micheles commented on openquake/hazardlib/tests/gsim/ghasemi_2009_test.py:
```diff
@@ -0,0 +1,29 @@
+class Ghasemi2009TestCase(BaseGSIMTestCase):
+    GSIM_CLASS = GhasemiEtAl2009
+
+    # Tables created from Ecxel calculation file of the proposed model 
```
Typo Ecxel -> Excel

FatemehAlsh replied:
it's been corrected.

mmpagani reviewed (approved):
Thank you. Overall this PR looks good to me. Please add your name here https://github.com/gem/oq-engine/blob/master/CONTRIBUTORS.txt and update this file https://github.com/gem/oq-engine/blob/master/debian/changelog

micheles commented:
Tests are red; please sure to refresh your branch with a `git pull origin/master`

FatemehAlsh commented:
> Tests are red; please make sure to refresh your branch with a `git pull origin/master`
a problem has occurred during updating. So I closed this pull request and I will open another one with all required changes discussed here. Thank you for your comments.

FatemehAlsh commented:
a problem has occurred during updating. So I closed this pull request and I will open another one with all required changes discussed here. Thank you for your comments.
